packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.tsx

```diff
@@ -97,7 +98,7 @@ export class HoverCard extends BaseComponent<IHoverCardProps, IHoverCardState> {
             {...getNativeProps(this.props, divProperties)}
             id={hoverCardId}
             trapFocus={!!this.props.trapFocus}
-            firstFocus={openMode === OpenCardMode.hotKey || openMode === OpenCardMode.hover}
+            firstFocus={!!this.props.setInitialFocus || openMode === OpenCardMode.hotKey}
```

Comment:
this.props.setInititialFocus is a Boolean with default props, so you don't need to !! it